tests/test_adjoint_jacobian.py

```diff
@@ -974,8 +1061,12 @@ def f(params1, params2):
         params1 = torch.tensor(0.3, requires_grad=True)
         params2 = torch.tensor(0.4, requires_grad=True)
 
-        h = 2e-3 if dev.R_DTYPE == np.float32 else 1e-7
-        tol = 1e-3 if dev.R_DTYPE == np.float32 else 1e-7
+        if dev._new_API:
+            h = 2e-3 if dev.c_dtype == np.complex64 else 1e-7
+            tol = 1e-3 if dev.c_dtype == np.complex64 else 1e-7
+        else:
+            h = 2e-3 if dev.R_DTYPE == np.float32 else 1e-7
+            tol = 1e-3 if dev.R_DTYPE == np.float32 else 1e-7
```

Comment:
Maybe a good candidate for a function too.

Reply:
Should we rename the property so that it matches the previous one?